app/mailers/custom_devise_mailer.rb

```diff
@@ -7,4 +8,8 @@ def confirmation_instructions(record, token, options = {})
     @confirmation_period = user_decorator.confirmation_period
     super
   end
+
+  def attach_images
+    attachments.inline['logo.png'] = File.read('app/assets/images/logo.png')
+  end
```

Comment:
The change was based on a recommendation from @amoose in https://github.com/18F/identity-private/issues/520#issuecomment-232415113.  There seems to be pros and cons to each method, the con to linking to image in the asset pipeline is that some email clients block external images by default: https://sendgrid.com/blog/embedding-images-emails-facts/.

We do have some credits from MailChimp to actually run some cross client tests on the emails, so I can do that with both techniques and see which performs better.